Chaincase/Views/VerifyMnemonicPage.xaml

```diff
@@ -19,16 +19,20 @@
                 <ColumnDefinition Width="*" />
 
             </Grid.ColumnDefinitions>
-            <Entry Text="{Binding Recall[0]}"
+            <Entry Text=""
+                   x:Name="Recall0"
```

Comment:
Do we need Text = ""? I think it's redundant, null is OK for our purposes

Reply:
I think it will work without. I'll try it

Reply:
OK. If you decide to, make the pr  :D